scorelab/Go-social
# Improved UI and image uploading in new post screen.

This pull request aims to improve the UI of the new post screen by fixing the add image and tag people buttons and placing the button at the bottom of the screen. Additionally, the post input is fixed, and image uploading is done using the right method of the 'react-native-image-picker'.

[new-post.webm](https://user-images.githubusercontent.com/91831606/227757138-ac86a1c6-74c5-41d6-b3ff-e341a972889a.webm)

**Test plan**
1. Go to the home screen and click on 'what's on you mind?' button.
2. Click on the input field and try to type.
3. Click on Add Image button and select the image.
4. Now click on the 'share' button on the top right
5. Check post is successfully published


**Code formatting**
The code follows the standard code formatting guidelines and is consistent with the existing codebase.

**Closing issues**
closes #178
